test/unit/test_float.c

```diff
@@ -558,9 +558,10 @@ bool f32_ieee_special_num_test(void) {
 }
 
 bool f32_ieee_rem_test(void) {
+	/* This test should return a different result in mod. */
 	RzFloat *a1 = rz_float_new_from_f32(4.0f);
 	RzFloat *b1 = rz_float_new_from_f32(1.5f);
-	RzFloat *expect1 = rz_float_new_from_f32(1.0f);
+	RzFloat *expect1 = rz_float_new_from_f32(-0.5f);
```

Comment:
this doesn't look correct

Reply:
So, according to my understanding, the remainder function works as follows.

```
rem(x, y) = x - rem_quot * y
```

where `rem_quot` is the result of `x/y` rounded toward the nearest integral value (with halfway cases rounded toward the even number). Hence, in our case, `x = 4.0` and `y = 1.5`.

```
rem_quot = round(4.0/1.5) = round(2.66666...) = 3
rem(x, y) = 4.0 - 3 * 1.5 = -0.5
```

Source: https://stackoverflow.com/questions/25734144/difference-between-c-functions-remainder-and-fmod

Reply:
Add a comment saying one is `rounded towards zero` and one `rounded nearest`

Reply:
or rename the test, because is not clear.

Reply:
You're right, I'll add a comment explaining this.